### PR TITLE
Created list of food & basic pages

### PR DESCRIPTION
**Pull request by pentabulge**
Created list of foods as well as basic pages for the corresponding items

### CODE REVIEW

File: pages/Monster-Jerky.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*"No longer hungry"*
```
Comment (TheBusyBiscuit) [inline pages/Monster-Jerky.md]: There is no need to repeat the lore on here, it doesn't really give much information about the Item.

File: pages/Monster-Jerky.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Monster Jerky is a [Food](Food.md) item added by Slimefun. When consumed, it removes any Hunger status effect and adds Saturation I for 0.25 seconds.
```
Comment (TheBusyBiscuit) [inline pages/Monster-Jerky.md]: Also just a tip:
Try to work with line-breaks to make content go on the next line:
The syntax is:
`<br>`

File: pages/Meat-Jerky.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+*"Extra saturating"*
```
Comment (TheBusyBiscuit) [inline pages/Meat-Jerky.md]: There is no need to repeat the lore on here, it doesn't really give much information about the Item.

File: pages/Diet-Cookie.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+*"A very lightweight cookie."*
```
Comment (TheBusyBiscuit) [inline pages/Diet-Cookie.md]: There is no need to repeat the lore on here, it doesn't really give much information about the Item.

File: pages/Juice.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Juice is a [Food](Food.md) item added by Slimefun.
```
Comment (TheBusyBiscuit) [inline pages/Juice.md]: This pages should rather be called "Juices" since it represents multiple types of Juices, not just one Juice.

**Comment by TheBusyBiscuit** [comment]
Hi, first of all: Thank you very much for the contribution.
But it looks like you haven't figured out how to create links to other pages yet, is that something I should add to the "Expanding the wiki" guide?

File: pages/Food.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+* [Beef Jerky](Meat-Jerky.md)
+* [Pork Jerky](Meat-Jerky.md)
+* [Chicken Jerky](Meat-Jerky.md)
+* [Mutton Jerky](Meat-Jerky.md)
+* [Fish Jerky](Meat-Jerky.md)
+* [Magic Sugar](Magic-Sugar.md)
+* [Monster Jerky](Monster-Jerky.md)
+* [Apple Juice](Juice.md)
+* [Carrot Juice](Juice.md)
+* [Melon Juice](Juice.md)
```
Comment (WalshyDev) [inline pages/Food.md]: I'd maybe link to the specific header in this file. https://link/file#Header

Comment (pentabulge) [reply pages/Food.md]: Thank you, I hadn't thought of it. The page doesn't have any headers yet but I will keep this in mind for the future.

**Pull request merged**